### PR TITLE
EPMRPP-37910 || Launch retry flag dao

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/dao/LaunchRepository.java [hunk 1]
```diff
@@ -56,4 +56,14 @@
...
 
+	/**
+	 * Checks if a {@link Launch} has items with retries.
+	 *
+	 * @param launchId Current {@link Launch#id}
+	 * @return True if has
+	 */
+	@Query(value = "SELECT exists(SELECT 1 FROM launch JOIN test_item ON launch.id = test_item.launch_id "
+			+ "WHERE launch.id = :launchId AND test_item.has_retries LIMIT 1)", nativeQuery = true)
+	boolean hasRetries(@Param("launchId") Long launchId);
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/dao/LaunchRepository.java]: tests

Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/dao/LaunchRepository.java]: done

File: src/test/java/com/epam/ta/reportportal/dao/suite/LaunchRepositoryTest.java [hunk 2]
```diff
@@ -225,6 +225,14 @@ public void identifyStatus() {
 		assertTrue(failed);
 	}
 
+	@Test
+	public void hasRetries() {
+
+		final boolean hasRetries = launchRepository.hasRetries(100L);
+		assertTrue(hasRetries);
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/dao/suite/LaunchRepositoryTest.java]: should be negative test as well to prove that method can return both of values

Comment (Yumfriez) [reply src/test/java/com/epam/ta/reportportal/dao/suite/LaunchRepositoryTest.java]: added

**Pull request merged**